openebs/cmd/openebs-provisioner/main.go

```diff
@@ -33,6 +35,9 @@ import (
 
 const (
 	provisionerName = "openebs.io/provisioner-iscsi"
+	// LeaderElectionKey represents ENV for disable/enable leaderElection for
+	// openebs-provisioner
+	LeaderElectionKey = "LEADER_ELECTION_ENABLED"
```

Comment:
should this be done for snapshot related controllers as well?

Reply:
Yes, only in  snapshot-provisioner. snapshot controller doesn't uses the controller lib